Yoast/wordpress-seo
# Removed configuration wizard block from dashboard

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Line out the Wordpress update notice with our content
* Fill out main window vertically
* Remove "First time SEO configuration" CSS

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* git checkout 14603-update-dashboard
* git reset --hard a35b4030a56616aafd822630f235a2ea832a528d
* Login into WP and head to the SEO dashboard.
* Check if left part of the dashboard stretches all the way to the right sidebar.
* Check if the configuration wizard block doesn't show up in the dashboard.


## UI changes

## Documentation

## Quality assurance


Fixes #14603
